src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+Microsoft Azure CLI 'acr' Command Module
+==================================
+
+This package has [not] been tested [much] with Python 2.7, 3.4 and 3.5.
+
+Commands to manage Azure container registries
```

Comment:
Is this something we typically do in packages (put the help text for all commands in the readme)? On the one hand it can be helpful, that is until the help text is changed and this doesn't get updated. I would also argue there's not much point repeating the global arguments in each entry.